ruby-bionic/3.0.x/hooks/post_push

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Any way we can DRY This out by having a single post_push script somewhere?

Reply:
According to the docs in Docker Hub, the hooks/ directory has to be found in the same directory as the Dockerfile. I've never done symbolic links in a git repo and don't know how docker hub would handle them in this case, but I'll give it a try and circle back around for a review.